db.go

```diff
@@ -636,7 +636,7 @@ func writeToFile(f *os.File, buf []byte) error {
 			return err
 		}
 	}
-	return nil
+	return f.Sync()
```

Comment:
Then we can just call `f.Write` once, remove the loop in `writeToFile`.

Reply:
AFAK, file IO such as `f.Write` will block the system thread (M in Go scheduler). I'm not sure whether a large `f.Write` will influence the performance due to a long blocking.

Reply:
Then please test it.

Reply:
BTW, what's the reason of change from `one write call` to `write loop`? 

Because I think call `f.Write` once and then call `f.Sync` is the same with call `f.Write` once with `O_SYNC`.

Reply:
The disk I/O has spike which adds latency to vlog write if we write a lot of data once, but if the write is not synced, then there seems no need to separate it.

Reply:
The I/O peak effect is found in a test where only one DB is used, after we separate to multiple DB, the performance maybe much different.